src/app/forms/forms.demo.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Component} from "@angular/core";
+
+@Component({
+    selector: "clr-forms-demo",
+    styles: [`.list-nav { columns: 3; margin-bottom: 20px; }`],
+    template: `
+        <h2>Forms</h2>
+        <ul class="list-nav">
+            <li><a [routerLink]="['./input-vertical']">Input Vertical</a></li>
+            <li><a [routerLink]="['./input-horizontal']">Input Horizontal</a></li>
+            <li><a [routerLink]="['./input-compact']">Input Compact</a></li>
+            <li><a [routerLink]="['./input-vertical-grid']">Input Vertical Grid</a></li>
+            <li><a [routerLink]="['./input-horizontal-grid']">Input Horizontal Grid</a></li>
+            <li><a [routerLink]="['./input-compact-grid']">Input Compact Grid</a></li>
+            <li><a [routerLink]="['./input-group']">Input Group</a></li>
```

Comment:
Do you think we should add a demo here which tests if popovers are being clipped inside of forms? I remember 2 main issues:
1. Angular tooltip component in forms in a modal
2. Datepicker in forms in a modal.

 but the Forms Refactor Epic has more details.

Reply:
Added modal examples, if you use a grid in a modal it still clips.

Reply:
Thank you!